### ISSUE TITLE
Not ideal GUI for multiple accounts, especially when sync conflicts occur

### ISSUE THREAD

**Issue by claell** [nextcloud/desktop#4152]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you want to have the same feature implemented.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Feature description
The UX for multiple users seems to be slightly flawed. At least I had a hard time figuring out what caused the Nextcloud tray icon to display a red x, while at the same time I did not see problems when clicking on it:

![grafik](https://user-images.githubusercontent.com/6171135/148588858-9cc19351-1608-4ec0-9366-dd6835e4dc92.png)

Only when I wanted to send logs, I saw by accident that the other connected account was causing the issues.

Of course, it was my fault that I did not check the other account and that I thought there might be a bug in the client responsible for this. However, this might also be a case of not ideal UX. Maybe one can improve that (for example, provide a merged feed of changes for all accounts or something like that).

Since there are already issues existing for the problems involved, this can also be used as a meta issue.

- [x] #3843
- [x] #4290

**Comment by LukeOwlclaw** [comment]
Right, UX for the tray icon is buggy. There are already a number of issues for that. [I created this one](https://github.com/nextcloud/desktop/issues/3298), which got closed due to inactivity. But there is also [this older issue](https://github.com/nextcloud/desktop/issues/2210) which was also closed without solution. But there are even more, e.g. [this even older (and currently open) issue addressing the icon problem](https://github.com/nextcloud/desktop/issues/619#issuecomment-708621154).

**Comment by claell** [comment]
Thanks for the hint. I have tried to link the already existing issues better. What remains is that this issue is not only about the sync icon (which is covered by #619) but also about the feed of changes etc.

**Comment by jancborchardt** [comment]
In the case of issues with another account, we should probably not display "All synced" in the summary, but rather:
> **Sync error with another account**
> username on cloud.example.com

And clicking on it would switch to that account.

**Comment by claell** [comment]
In general, I was thinking about improving the behavior for multiple accounts. For example, one could make switching between accounts easier to discover. Or offer a merged feed that contains all events for all accounts. Or offer different tray icons per account (this might be a good option, especially if one uses many accounts, where it is not really possible to reflect the state of each account with a single tray icon anymore).

**Comment by Rello** [comment]
Closing as resolved after an automated backlog review. The tray redesign in merged PR #9977 provides explicit per-account surfaces and status, while merged PR #10350 moved activity and conflict handling into a dedicated window. These changes address the multi-account discoverability problem described by this meta issue.
